src/main/kotlin/com/terraformation/backend/util/Constants.kt

```diff
@@ -2,3 +2,6 @@ package com.terraformation.backend.util
 
 /** Number of square meters in a hectare. */
 const val SQUARE_METERS_PER_HECTARE: Double = 10000.0
+
+/** Lead time in minutes for module event STARTING_SOON and notifications state */
+const val MODULE_EVENT_NOTIFICATION_LEAD_TIME_MINS: Long = 15
```

Comment:
Could make this a `Duration` instead since it's only ever used to construct one. It can't be `const` in that case, but that's fine.

Also, maybe move this under the accelerator package since it's only relevant for accelerator functionality.